README.markdown

```diff
@@ -95,6 +95,65 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+**NOTE**: There is a bug currently, that causes FakeFS to fail with the following error:
+```
+TypeError:
+  superclass mismatch for class File
+```
+
+To prevent this, you'll need to require 'pp' *BEFORE* you require 'fakefs/spec_helpers' in your
+spec_helpers file. Like this:
+
+```ruby
+require 'pp'
+require 'fakefs/spec_helpers'
```

Comment:
I don't know if having some history helps, but this has come up before.  In issue https://github.com/fakefs/fakefs/issues/215 I mentioned that I used the solution from https://github.com/fakefs/fakefs/issues/99.  For what it's worth, I've been doing the `require 'pp'` after the `require 'fakefs/spec_helpers'` - I think it's just important to ensure the `require 'pp'` happens before any calls that actually use `fakefs`.